ui/brave_ads/message_popup_view.cc

```diff
@@ -87,20 +85,31 @@ MessagePopupView::MessagePopupView(const Notification& notification) :
   views::Widget::InitParams params(views::Widget::InitParams::TYPE_POPUP);
   params.type = views::Widget::InitParams::TYPE_WINDOW_FRAMELESS;
   params.z_order = ui::ZOrderLevel::kFloatingWindow;
-  const gfx::Size& screen_size =
-      display::Screen::GetScreen()->GetPrimaryDisplay().size();
+
+  const gfx::Rect& screen_visible_frame = GetVisibleFrameForPrimaryDisplay();
+
+  const int x = screen_visible_frame.width()
+      - (kPopupBaseWidth + kPopupPadding);
+
+  const int width = kPopupBaseWidth;
+  const int height = kPopupBaseHeight + GetBodyHeight(notification.message());
+
 #if defined(OS_WIN)
-  const uint64_t screen_y = screen_size.height() - kPopupYDeltaWin;
+  const int screen_y = screen_visible_frame.height() - (height + kPopupPadding);
```

Comment:
```suggestion
  const int y = screen_visible_frame.y() - (height + kPopupPadding);
```

Reply:
@yachtcaptain23 thanks, changed but different than the above recommendation. Waiting for Windows build to test.